### DISCUSSION TITLE
Connection lost

### DISCUSSION THREAD

**Discussion by navnforfun** [elmoiv/elmocut#69] [Q&A]
![image](https://github.com/elmoiv/elmocut/assets/147603262/fc3582d8-226e-4a84-8420-20c38d87e958)
How can i fix? I use npcap 1.7.3 and elmocut 1.7

**Comment by elmoiv** [comment]
Try downgrading to v1.10 from here:
https://nmap.org/npcap/dist/npcap-1.10.exe

**Reply by navnforfun** [reply]
I have been downgraded npcap to v1.1 but this not help

**Comment by elmoiv** [comment]
Can you check your Windows language and send me screenshot of the adapters you're connected to from elmoCut settings

**Reply by navnforfun** [reply]
![image](https://github.com/elmoiv/elmocut/assets/147603262/f4171660-e9c4-4c69-a523-21646ce37156)

**Reply by elmoiv** [reply]
Send me screenshot for interfaces list from elmoCut settings
